tests/common/utilities.py

```diff
@@ -661,3 +661,15 @@ def update_environ(*remove, **update):
         env.update(to_restore)
         for k in to_removed:
             env.pop(k)
+
+
+def get_image_type(duthost):
+    """get the SONiC image type
+    It might be public/microsoft/...or any other type.
+    Different vendors can define their different types by checking the specific information from the build image.
+    Args:
+        duthost: AnsibleHost instance for DUT
+    Returns: image type. Str. It should be the right key in DEFAULT_SSH_CONNECT_PARAMS in tests/common/constants.py
```

Comment:
Improve the help text as below?

The returned image type string will be used as a key of map DEFAULT_SSH_CONNECT_PARAMS defined in `tests/common/constants.py` for looking up default credential for this type of image.

Reply:
Modified.